src/lib/tableland-calls.ts

```diff
@@ -128,13 +130,19 @@ async function validateWriteQuery(
   return camelCaseKeys(json.result);
 }
 
-async function read(this: Connection, query: string): Promise<ReadQueryResult> {
+async function read(
+  this: Connection,
+  query: string,
+  options: ReadOptions = {}
+): Promise<ReadQueryResult> {
+  if (!options.output) options.output = "table";
   const message = await GeneralizedRPC.call(this, "runReadQuery", {
     statement: query,
+    ...options,
   });
   const json = await SendCall.call(this, message);
 
-  return camelCaseKeys(json.result.data);
+  return json.result.data;
```

Comment:
Good callout,.`camelCaseKeys` is used elsewhere, and removing is worth considering.  The original idea was that in Java/typescript variable naming convention is more commonly lower camel case than snake case. That might have been a short sided idea, but we use this currently to ensure the method calls return the same properties regardless of if the relay is used or not.  
My opinion is that it makes sense to continue to convert to lower camel case as long as we use a relay. This is because ethers.js uses lower camel case, and because we have to do some kind of conversion we might as well go with the LCC since that seems to be more JS friendly.  The LCC vs SC is definitely just a preference, so I can definitely be talked out of this if it doesn't make sense to others.
@carsonfarmer has a TPD in Notion for better aligning of apis across everything which feels related to this. https://www.notion.so/textile/Tableland-public-APIs-across-platforms-are-inconsistent-33d1fbac20144d97b88ec889458f7f79